server/debian/atomiadns-database.postinst

```diff
@@ -664,6 +664,12 @@ if [ $retval != 0 ]; then
 				recreate_procedure "unassigntsigkey"
 			fi
 
+			if [ "$schema_version" -lt 94 ]; then
```

Comment:
You need to bump the version at the start of this file [here](https://github.com/atomia/atomiadns/blob/797868293e6ba4d2cb1b0ff4c44e32b117f866fc/server/debian/atomiadns-database.postinst#L299)

Reply:
Resolved.